internal/repository/token_repository.go

```diff
@@ -0,0 +1,44 @@
+package repository
+
+import (
+	"context"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"example.com/tfgrid-kyc-service/internal/models"
+)
+
+type MongoTokenRepository struct {
+	collection *mongo.Collection
+}
+
+func NewMongoTokenRepository(db *mongo.Database) TokenRepository {
+	return &MongoTokenRepository{
+		collection: db.Collection("tokens"),
+	}
+}
+
+func (r *MongoTokenRepository) SaveToken(ctx context.Context, token *models.Token) error {
+	token.CreatedAt = time.Now()
```

Comment:
Maybe to be UTC instead

Reply:
Bson decoder converts this to and from UTC by default while dealing with the DB. It just works as it is.

Reply:
Where did you configure the time.Local to be time.UTC?  afaik mongo uses time.Local